ServerUtil.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const http = require('http');
+const https = require('https');
+const pubsub = require('pubsub-js');
+const lib = require('./lib');
+const CONST = require('./lib/constants');
+const logger = lib.logger;
+
+class HttpServer {
+  static startServer(app) {
+    const port = app.get('port');
+    const title = app.get('title');
+    const ssl = app.get('ssl');
+    const type = app.get('type');
+    const server = ssl ? https.createServer(ssl, app) : http.createServer(app);
+    server.on('error', onerror);
+    server.on('listening', onlistening);
+    server.listen(port);
+
+    function onerror(err) {
```

Comment:
can you modify all the loggers in this function to concatenate in the standard way we do in other places ?

Like - https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/136/files#diff-839d93c5f783939b13c6ae0dee13f50bR41

Reply:
Done.